<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="JINWX-PC" name="com.test.suit.discover.DeviceList_RmeterTests" tests="4" failures="0" timestamp="08 六月 2018 10:14:48 GMT" time="0.134" errors="1">
  <testcase name="noasso" time="0.134" classname="com.test.suit.discover.DeviceList_RmeterTests">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.test.testdata.DeviceListUrl.setNAInfo(DeviceListUrl.java:74)
at com.test.suit.discover.DeviceList_RmeterTests.noasso(DeviceList_RmeterTests.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- noasso -->
  <testcase name="emList" classname="com.test.suit.discover.DeviceList_RmeterTests">
    <skipped/>
  </testcase> <!-- emList -->
  <testcase name="treeList" classname="com.test.suit.discover.DeviceList_RmeterTests">
    <skipped/>
  </testcase> <!-- treeList -->
  <testcase name="rmList" classname="com.test.suit.discover.DeviceList_RmeterTests">
    <skipped/>
  </testcase> <!-- rmList -->
</testsuite> <!-- com.test.suit.discover.DeviceList_RmeterTests -->
